### DISCUSSION TITLE
filter option for "import is not used"

### DISCUSSION THREAD

**Discussion by HPWebdeveloper** [imanghafoori1/laravel-microscope#203] [Ideas]
Some of the imports are originally comes with laravel classes. like Illuminate\Contracts\Auth\MustVerifyEmail

and it says: "import is not used"

It would be great if there is a flag to ignore those come with laravel and only show those we add.

Thanks!

**Comment by imanghafoori1** [comment]
you just comment it out if you do not really use it.
